pastis/temporal_analysis/harris_mode.ipynb

```diff
@@ -0,0 +1,2335 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "30a87e9f",
+   "metadata": {},
+   "source": [
+    "## This notebook contains all the scripts used to do segment level thermal sensitivity analysis and produce plots all plots required for presentation, report or article. The code block which are commented aren't necessary to do full e2e analysis, but are used to support report/article."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9cb4388c",
+   "metadata": {},
+   "source": [
+    "###  Import neccessary python libraries and pastis functions"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c90040ca",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import time\n",
+    "import hcipy\n",
+    "import scipy\n",
+    "from scipy.interpolate import griddata\n",
+    "import glob\n",
+    "import numpy as np\n",
+    "from PIL import Image\n",
+    "from astropy.io import fits\n",
+    "import astropy.units as u\n",
+    "import pandas as pd\n",
+    "import exoscene.image\n",
+    "import exoscene.star\n",
+    "import exoscene.planet\n",
+    "from exoscene.planet import Planet\n",
+    "import matplotlib.pyplot as plt\n",
+    "from matplotlib.colors import TwoSlopeNorm\n",
+    "import matplotlib.gridspec as gridspec\n",
+    "from pastis.temporal_analysis.close_loop_analysis import req_closedloop_calc_batch\n",
```

Comment:
You... just made the effort of sorting your imports. Why did you put this smack in the middle of everything without following your own ordering? I am just trying to understand.